ishimwebernard/bernard_portfolio
# Ch api setupserver

#### What does this PR do?
[what does this PR do?]
>Setting the database up
>Created the dummy routes
#### Description of Task to be completed
[the tasks that you completed on the feature]
>
#### How should this be manually tested?
>Clone this repo at this branch
>Use cd server to go to the server directory
>use npm start to start the server
>use postman with the port address 3000 and to the route named post to see if the root endpoint is reached
#### Any background context you want to provide?
>N/A
#### What are the relevant Trello story?
>https://trello.com/c/PfbX0w7q
#### Screenshots (if appropriate)
>
